daemon/containerd/image_list.go

```diff
@@ -91,12 +91,26 @@ func (i *ImageService) Images(ctx context.Context, opts types.ImageListOptions)
 			return nil, err
 		}
 
+		repoTag := "<none>:<none>"
+		repoDigest := "<none>@<none>"
```

Comment:
I cringe every time I see these on the daemon side 😞 (these should probably never have been part of the API, but something on the CLI side when presenting).

It looks like we already have this for the existing `Images()` service, so not something to fix for this PR, but we should probably have a tracking issue to;

- (If possible) move this to the API (something like `if len(RepoDigests) == 0` set it to `<none>@<none>`)
- Deprecate the magic `<none>` values (if the code is in the API, we can more easily make it API version dependent; only add the magic values for API < version.X )
- Add code on the CLI side to fill in the magic values when presenting.


It looks like there's already _some_  handling on the CLI side, but there's more changes needed; https://github.com/docker/cli/blob/5f9c58ffa099af148e0285926aa9be5acd25ae77/cli/command/formatter/disk_usage.go#L157-L159